Source/WebCore/dom/PopoverData.cpp

```diff
@@ -42,4 +42,13 @@ Ref<ToggleEventTask> PopoverData::ensureToggleEventTask(Element& element)
     return *m_toggleEventTask;
 }
 
+void PopoverData::setInvoker(HTMLElement* popover, HTMLElement* invoker)
+{
+    if (RefPtr oldInvoker = m_invoker.get(); oldInvoker && !invoker)
+        oldInvoker->setInvokedPopover(nullptr);
+    m_invoker = invoker;
+    if (RefPtr newInvoker = m_invoker.get())
+        newInvoker->setInvokedPopover(popover ? RefPtr { popover } : nullptr);
+}
```

Comment:
Passing both the popover and the invoker to this function is kinda yucky. @rniwa suggested offline we could keep make this method only do `m_invoker = invoker;` and create a new `HTMLElement::setInvoker()` method that sets the invoker & does the extra logic of changing `setInvokedPopover` and use that at the call sites.

We should also make `PopoverData::setInvoker` private/protected so people don't call the wrong method.

Reply:
PopoverData::setInvoker can't be private or protected because else it cant be called inside HTMLElement::setInvoker.
Otherwise I've done the suggested changes.